### ISSUE TITLE
Broken Fourier model: "Cannot find specified frequency in DCDFT result" error

### ISSUE THREAD

**Issue by mpyat2** [AAVSO/VStar#278]
To reproduce:
1) File->New Star from AAVSO database..., load PMAK V1 all times.
2) Analyse->Fourier model: Johnson V, Number of Periods = 1, P= 0.4028, OK.
3) You will get an error:
![image](https://user-images.githubusercontent.com/44919945/170840462-06fe07e6-d0ff-4b34-990f-fdbe98867e0e.png)

**Comment by dbenn** [comment]
@mpyat2, `findIndexOfFrequency()` in `PeriodAnalysisDerivedMultiPeriodicModel` is invoked for FWHM and Standard Error of the Frequency, which in turn invokes `algorithm.getResultSeries()`. This will return an empty list unless a period analysis (DCDFT) has been carried out. In the case of the Fourier model plugin, there is no period analysis so no result data.

Standard Error of the Semi-amplitude, surprisingly, does not rely upon a period analysis but by itself, I'm not sure this is useful.

My suggestion is therefore, rather than allowing the exception to propagate, it should be caught and no Uncertainty tab will be created in the Models dialog where a period analysis result is available.

So basically, at the moment, if the user wants to get period error information, a DCDFT will have to be performed, not just a Fourier model.

Perhaps in a future release we can find another approach to this problem or use another approach to period error.

**Comment by mpyat2** [comment]
@dbenn , I agree, that the exception should be caught and the Model dialog should be available without the Uncertainty tab if the uncertainty information is not available, or the Uncertainty tab could be displayed with a short message pointing to the requirement of the period analysis to get the uncertainty.
There are cases where the Fourier model is useful even without detailed period info.

**Comment by dbenn** [comment]
I like your idea of adding a message to the Uncertainty tab. I'll do this right now.

**Comment by dbenn** [comment]
I'm adding the text: A period analysis must be carried out for uncertainty information to be computed.

**Comment by mpyat2** [comment]
@dbenn , great
